src/ef-coap.c

```diff
@@ -178,11 +178,11 @@ static int options_fill_defaults(struct frame *f, int stack_idx) {
     field_t *fnum = find_field(hdr, "num");
     field_t *fval = find_field(hdr, "val");
 
-    if (!fnum) {
+    if (!fnum->val || !fval->val) {
         return 0;
     }
 
-    if (fval){
+    if (fval) {
```

Comment:
This change is just a style change so it is not needed in this commit.

Reply:
Yes okay , Updated now .
Thank you